doc/advanced-configuration/outgoing-connections.md

```diff
@@ -1,177 +1,92 @@
-{redis, <<"b.com">>, default, PoolOpts, ConnOpts},
-{redis, <<"c.eu", default, PoolOpts, ConnOptsForDomain2},
-{redis, <<"d.eu">>, default, PoolOpts, ConnOpts}
-```
+#### `outgoing_pools.*.*.call_timeout`
+* **Syntax:** positive integer
+* **Default:** 5000
+* **Example:** `call_timeout = 5000`
 
-## RDBMS connection setup
+# Connection options
 
-An example RDBMS configuration inside `outgoing_pools` may look like this:
+Options specific to a pool connection are defined in a subsection starting with `[outgoing_pools.*.*.connection]`.
+For example:
 
-```erlang
-{outgoing_pools, [
- {rdbms, global, default, [{workers, 5}],
-  [{server, {mysql, "localhost", 3306, "mydb", "user", "passwd"}}]}
-]}.
 ```
+[outgoing_pools.rdbms.default]
+  scope = "global"
+  workers = 5
 
-This configuration will create a default, global pool of 5 connections to a mysql database.
-We might also want to add a dedicated pool for a specific host:
-
-```erlang
-{outgoing_pools, [
- {rdbms, global, default, [{workers, 5}],
-  [{server, {mysql, "localhost", 3306, "mydb", "user", "passwd"}}]},
- {rdbms, "myhost.com", default, [{workers, 3}],
-  [{server, {mysql, "localhost", 3306, "mydb", "user", "passwd"}}]}
-]}.
+  [outgoing_pools.rdbms.default.connection]
+  ...
 ```
 
-Please remember that SQL databases require creating a schema.
-See [Database backends configuration](./database-backends-configuration.md) for more information.
-Also see [Advanced configuration](../Advanced-configuration.md) for additional options that influence RDBMS connections.
-Currently all pools must use the same RDBMS type (e.g. `mysql`, `pgsql`).
-
-### Connection options
-
-* **server**
-    * **Description:** SQL DB connection configuration. Currently supported DB types are `mysql` and `pgsql`.
-    * **Syntax:** `{server, {Type, Host, Port, DBName, Username, Password}}.` **or** `{server, "<ODBC connection string>"}`
-    * **Default:** `undefined`
-
-* **keepalive_interval**
-    * **Description:** When enabled, will send `SELECT 1` query through every DB connection at given interval to keep them open.
-    This option should be used to ensure that database connections are restarted after they became broken (e.g. due to a database restart or a load balancer dropping connections).
-    Currently, not every network related error returned from a database driver to a regular query will imply a connection restart.
-    * **Syntax:** `{keepalive_interval, IntervalSeconds}.`
-    * **Example:** `{keepalive_interval, 30}.`
-    * **Default:** `undefined`
-
-#### MySQL and PostgreSQL SSL connection setup
-
-In order to establish a secure connection with a database, additional options must be passed in the `server` tuple.
-Here is the proper syntax:
+## RDBMS options
 
-`{server, {Type, Host, Port, DBName, Username, Password, SSL}}.`
+#### `outgoing_pools.rdbms.*.driver`
+* **Syntax:** a supported driver, currently `"pgsql"`, `"mysql"` or `"odbc"`
```

Comment:
Not sure if needed here, but in most places we were stating the type explicitly, i.e.
```suggestion
* **Syntax:** string, one of `"pgsql"`, `"mysql"` or `"odbc"` (a supported driver)
```